packages/design-system-tailwind-preset/src/typography.ts

```diff
@@ -0,0 +1,164 @@
+export const typography = {
+  fontSize: {
```

Comment:
These are the current equivalent of what we would consider component tokens. These could be simplified in future with the Typography Figma variant work which we would do at the component level. There are some font weight changes at the variant level which is easiest to define at each variable.